transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,73 @@
+{{
+    config({
+        "tags":"hourly",
+    })
+}}
+
+WITH MM_TELEMETRY_PROD AS 
+(SELECT _dbt_source_relation	  AS _dbt_source_relation	
+ , context_page_referrer          AS context_page_referrer
+ , channel                        AS channel
```

Comment:
Quite a few of these columns are shared across all Rudderstack events. It's extremely likely that we'll need them again when extracting data for other events. Perhaps we can find a way to abstract this so that we don't repeat it over code.

Adding a bit more info, each event's columns are divided in three groups:
- Standard rudderstack columns (i.e. `id`, `anonymous_id`, `timestamp`, `received_at` etc). A detailed list is available [here](https://www.rudderstack.com/docs/destinations/warehouse-destinations/warehouse-schema/#table-add_to_cart).
- Standard column conventions from Mattermost's servers. Examples are `user_id`, `user_actual_id`, `category`, `type`. The tricky part with these columns is they don't exist in every event table.
- Event-specific properties. 

The first two groups can be abstracted to a macro (i.e. `rudderstack_columns()`). This will increase readability of staging models.